src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/DefaultReadDfMethods.kt

```diff
@@ -9,6 +9,28 @@ private const val cast = "cast"
 private const val verify = "verify" // cast(true) is obscure, i think it's better to use named argument here
 private const val readCSV = "readCSV"
 private const val readJson = "readJson"
+private const val readArrow = "readArrow"
+
+public class DefaultReadArrowMethod(private val path: String?) : DefaultReadDfMethod {
+    override fun toDeclaration(markerName: String, visibility: String): String {
+        return buildString {
+            if (path != null) {
+                append("""        ${visibility}const val defaultPath: String = "${path.escapeStringLiteral()}"""")
+                appendLine()
+            }
+            val defaultPathClause = if (path != null) " = defaultPath" else ""
+            append(
+                """
+                |        ${visibility}fun $readArrow(path: String$defaultPathClause, $verify: Boolean? = null): DataFrame<$markerName> { 
+                |            val df = DataFrame.$readArrow(path)
```

Comment:
Add import for readArrow method here:
https://github.com/Kotlin/dataframe/blob/9d28735112c034b7d56791660a19b8746fcf44c3/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/codeGen/CodeGeneratorImpl.kt#L291
It probably should be that each read method declares its own imports, but i think supporting it would require a lot of changes. So let's leave it out of scope of this PR :)